manager/allocator/networkallocator/networkallocator.go

```diff
@@ -191,6 +191,14 @@ func (na *NetworkAllocator) ServiceAllocate(s *api.Service) (err error) {
 
 vipLoop:
 	for _, eAttach := range s.Endpoint.VirtualIPs {
```

Comment:
Calling the vip `eAttach` is very confusing. It took me some time to understand the logic inside the loop because I was assuming `eAttach` was anything but a VIP.

This is existing code and not related to this PR. We should rename  this after this later.